assets/airports/rjaa.json

```diff
@@ -1,1158 +1,2057 @@
-                ["dal", 17],
-                ["uae",  2],
-                ["etd",  1],
-                ["eva",  2],
-                ["gia",  2],
-                ["hal",  1],
-                ["ibe",  1],
-                ["jal", 34],
-                ["jjp", 12],
-                ["klm",  2],
-                ["kal",  5],
-                ["dlh",  3],
-                ["mas",  5],
-                ["qfa",  1],
-                ["qtr",  1],
-                ["sbi",  2],
-                ["sas",  2],
-                ["sco",  3],
-                ["sia",  2],
-                ["sjo",  4],
-                ["swr",  2],
-                ["thy",  1],
-                ["ual", 10],
-                ["fdx",  4],
-                ["nca", 16],
-                ["sia/cargo", 2],
-                ["ups",  4]
+				["amu", 14],
+				["ana", 57],
+				["apj", 56],
+				["cal", 35],
+				["cpa", 72],
+				["crk", 12],
+				["csz", 1],
+				["eva", 28],
+				["hke", 30],
+				["jjp", 49],
+				["jal", 56],
+				["nca", 7],
+				["pac", 2],
+				["sky", 2],
+				["tgw", 13],
+				["ttw", 18]
             ]
-        },
-        {
+		},
+		{
             "origin": "RJAA",
             "destination": "",
             "category": "departure",
-            "route": "RJAA.SAKURA4.GOTEN",
-            "altitude": null,
-            "speed": null,
+            "route": "RJAA16R.REDEK2.REDEK.Y60.TEKEL",
+            "altitude": "",
+            "speed": "",
             "method": "random",
-            "rate": 1.44,
+            "rate": 1.49,
             "airlines": [
-                ["afl",  1],
-                ["amx",  1],
-                ["aca",  3],
-                ["cca",  7],
-                ["afr",  2],
-                ["aic",  1],
-                ["aza",  2],
-                ["ana", 54],
-                ["aal",  3],
-                ["aua",  1],
-                ["baw",  1],
-                ["cpa",  3],
-                ["ces",  3],
-                ["csn",  7],
-                ["dal", 17],
-                ["uae",  2],
-                ["etd",  1],
-                ["eva",  2],
-                ["gia",  2],
-                ["hal",  1],
-                ["ibe",  1],
-                ["jal", 34],
-                ["jjp", 12],
-                ["klm",  2],
-                ["kal",  5],
-                ["dlh",  3],
-                ["mas",  5],
-                ["qfa",  1],
-                ["qtr",  1],
-                ["sbi",  2],
-                ["sas",  2],
-                ["sco",  3],
-                ["sia",  2],
-                ["sjo",  4],
-                ["swr",  2],
-                ["thy",  1],
-                ["ual", 10],
-                ["fdx",  4],
-                ["nca", 16],
-                ["sia/cargo", 2],
-                ["ups",  4]
+				["aic", 2],
+				["ana", 84],
+				["cca", 18],
+				["ces", 17],
+				["cqh", 7],
+				["csc", 1],
+				["csn", 9],
+				["cxa", 5],
+				["dkh", 7],
+				["edt", 7],
```

Comment:
This airline does not currently exist, and is causing an error. Please add the airline to this PR.